packages/frontend/src/components/accounts/create/implicit_account/CreateImplicitAccount.js

```diff
@@ -94,21 +99,34 @@ export default ({
         <>
             <StyledContainer className='border small-centered'>
                 <h3><Translate id='account.createImplicitAccount.title' /></h3>
-                {/* <div className='flex-center-center'>
+                {<div className='flex-center-center'>
                     {['30', '50', '100', ''].map((amount) => (
                         <BuyButton
                             key={`${amount}`}
                             amountUSD={amount}
-                            onClickBuyButton={onClickBuyButton}
-                            moonpayIsAvailable={moonpayIsAvailable}
+                            onClickBuyButton={() => {
+                                window.open(
+                                    buildUtorgPayLink(implicitAccountId, undefined, amount, 'USD'),
+                                    '_blank'
+                                );
+                            }}
                         />
                     ))}
-                </div> */}
+                </div>}
                 {/*
-                TEMPORARILY DISABLE MOONPAY PRE-SPECIFIED AMOUNT BUTTONS
+                TEMPORARILY DISABLE MOONPAY PRE-SPECIFIED AMOUNT BUTTONS,
                 WHILE MOONPAY 'baseCurrencyAmount' IS BROKEN
                 ISSUE: https://github.com/near/near-wallet/issues/2408
+
+                ENABLED UTORG
                 */}
+                <FormButton
+                    linkTo={buildUtorgPayLink(implicitAccountId)}
+                    color='black'
```

Comment:
Would we want to have the same loading condition here as for the moonpay button below using the `sending` prop?

Reply:
I added implicitAccountId condition in sending prop, but it does nothing. Because implicitAccountId available immediately.